README.md

```diff
@@ -22,92 +23,92 @@ documentation, original graphics and other material are released under the
 Licenses of all packages used by Spine Toolbox are listed in the Spine Toolbox User 
 Guide.
 
+## Installation
 
-## Official releases
+Three options for installing Spine toolbox exist: as an application within
+an existing Python installation, using stand-alone installation package and using 
+the source files directly (for developers).
 
-Release versions of Spine Toolbox can be found 
-[here](https://drive.google.com/drive/folders/1t-AIIwRMl3HiYgka4ex5bCccI2gpbspK)
-(only available for 64-bit Windows for now). Download the latest version, install and
-run `spinetoolbox.exe`.
+### With standard Python installation
 
-## Development version
+Spine Toolbox is available on [The Python Package Index (PyPI)](https://pypi.org/project/spinetoolbox/).
+This option is the recommended way to get the most recent release version but a
+working Python installation is also required. 
 
-The development version of Spine Toolbox is in the master branch on this repository, and it has all the latest features and bug-fixes.
-Please note that this version has passed automated testing, but has not been completely manually tested.
+---
+**NOTE:** If you already have Spine toolbox installed globally using pip, please first 
+uninstall using `pip uninstall spinetoolbox`.
 
-### Installation
+---
 
-1. Do you have Conda? If yes, go directly to step 2. Otherwise, install either [Miniconda](https://docs.conda.io/en/latest/miniconda.html)
-(recommended) or [Anaconda](https://www.anaconda.com/distribution/) (good alternative).
+1. Install Python 3.8. On Windows systems, the easiest way is through **Microsoft Store** 
+or [Python.org](https://www.python.org/downloads/windows/). On OS X, get 
+[the latest release at Python.org](https://www.python.org/downloads/mac-osx/). 
+Most Linux distributions have Python preinstalled, but please check which version 
+you have and update if necessary.
 
-2. Open [Anaconda Prompt](https://docs.anaconda.com/_images/win-anaconda-prompt1.png).
+2. Make sure you have the latest version of pip installed:
 
-3. Create a new environment by typing:
+        python3 -m pip install --upgrade pip
```

Comment:
```
C:\>python3 -m pip install --upgrade pip
'python3' is not recognized as an internal or external command,
operable program or batch file.
```

`python3` is a linux thing. How about just `python`?

Reply:
On my Windows, calling `python3` fires up MS Store to install Python. I thought it would be a quite safe option. I can change that to just `python`. On some systems that might launch some system-default Python 2.7 and that’s not what we want...

Reply:
I added a note that, on some systems one might need to call `python3` instead.

Reply:
Ok, I know what happened. There are these *App execution aliases* in win10 settings. There's a default setting when typing python.exe or python3.exe into command prompt. I turned these settings off some time ago. If `python3 -m pip install --upgrade pip` works with the default windows settings, then this was not actually an issue (sorry).

Reply:
It is good now. This is resolved.